File: class14/python/AP1411-PositivoNegativoZero.py
# PositivoNegativoZero — Aula 14
# Classifica um número como positivo, negativo ou zero

num = float(input("Digite um número: "))
if num > 0:
    print(f"{num} é POSITIVO")
elif num < 0:
    print(f"{num} é NEGATIVO")
else:
    print("O número é ZERO")

File: class14/cpp/AP1411-PositivoNegativoZero.cpp
#include <iostream>
int main() {
    double num;
    std::cout << "Digite um número: "; std::cin >> num;
    if (num > 0) std::cout << num << " é POSITIVO" << std::endl;
    else if (num < 0) std::cout << num << " é NEGATIVO" << std::endl;
    else std::cout << "O número é ZERO" << std::endl;
    return 0;
}